src/Shared/LayoutRenderers/AspNetSessionValueLayoutRenderer.cs

```diff
@@ -104,8 +104,8 @@ protected override void DoAppend(StringBuilder builder, LogEventInfo logEvent)
             {
                 if (!reEntry.IsLockAcquired)
                 {
-                    InternalLogger.Error(
-                        $"Reentrant log event detected. Logging when inside the scope of another log event can cause a StackOverflowException. LogEventInfo.Message:{logEvent.Message}");
+                    InternalLogger.Debug(
+                        $"Reentrant log event detected. Logging when inside the scope of another log event can cause a StackOverflowException.");
```

Comment:
Please also remove `$"` since not string-interpolation.

Reply:
done